docs/overview.yaml

```diff
@@ -97,7 +97,7 @@ General algebra:
     associative algebra over a commutative ring: 'Algebra'
     the category of algebras over a ring: 'AlgebraCat'
     free algebra of a commutative ring: 'FreeAlgebra'
-    tensor product of algebras: 'Algebra.TensorProduct.instAlgebraTensorProductToAddCommMonoidToNonUnitalNonAssocSemiringToNonAssocSemiringToAddCommMonoidToNonUnitalNonAssocSemiringToNonAssocSemiringToModuleToModuleInstSemiringTensorProductToAddCommMonoidToNonUnitalNonAssocSemiringToNonAssocSemiringToAddCommMonoidToNonUnitalNonAssocSemiringToNonAssocSemiringToModuleToModule'
+    tensor product of algebras: 'Algebra.TensorProduct.instAlgebraTensorProductToAddCommMonoidToNonUnitalNonAssocSemiringToNonAssocSemiringToAddCommMonoidToNonUnitalNonAssocSemiringToNonAssocSemiringToModuleToModuleInstSemiring'
```

Comment:
Could you change that instance name as well? I would guess that it is a good idea to at least change all instance names that appear explicitly somewhere in mathlib.